gsadjust/pyqt_models.py

```diff
@@ -1635,6 +1635,16 @@ class DatumTableModel(QtCore.QAbstractTableModel):
         N_SETS: '# sets',
     }
 
+    _attrs = { # From column constants to object attributes, for setting.
```

Comment:
This is an example of using the same "routing" approach for setting data. Here we define a dictionary keyed by the column constants with values of the attribute on the object to set, and the type to convert the data to.  So for example when setting column DATUM_STATION we do the equivalent of `setattr(self, 'station', str(value))`